### PR TITLE
10.0 website multi company rule for site edditing

### PR DESCRIPTION
**Pull request by iledarn**

### CODE REVIEW

File: website_multi_company/controllers/main.py [hunk 1]
```diff
@@ -52,3 +52,14 @@ def index(self, **kw):
...
         return self.page(page)
+
+    @http.route()
+    def page(self, page, **opt):
+        response = super(WebsiteExtended, self).page(page, **opt)
+        uid = request.session.uid
+        if uid:
+            user = request.env['res.users'].browse(uid)
+        response.qcontext.update({
+            'editable': uid and user and user.company_id == request.website.company_id,
```
Comment (yelizariev) [inline website_multi_company/controllers/main.py]: ``uid`` is always truthy here

File: website_multi_company/doc/changelog.rst [hunk 2]
```diff
@@ -1,3 +1,8 @@
+`1.2.4`
+-------
+
+- **ADD:** Product with no company defined is allowed for all websites
```
Comment (yelizariev) [inline website_multi_company/doc/changelog.rst]: No note about access rights for editors

File: website_multi_company_sale/models/product_template.py [hunk 3]
```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
+
+from odoo import api, fields, models, _
+from odoo.exceptions import ValidationError
+
+
+class ProductTemplate(models.Model):
+    _inherit = "product.template"
+
+    website_ids = fields.Many2many('website', string='Allowed websites',
```
Comment (yelizariev) [inline website_multi_company_sale/models/product_template.py]: website_multi_company_sale has to have changelog and version updates too

**Comment by iledarn** [comment]
Superseded by https://github.com/it-projects-llc/website-addons/pull/297

**Pull request closed without merging**